python/vespa/vespa/application.py

```diff
@@ -76,6 +77,22 @@ def query(
             r = post(self.search_end_point, json=body)
             return VespaResult(vespa_result=r.json())
 
+    def feed_data_point(self, schema: str, data_id: str, fields: Dict) -> Response:
+        """
+        Feed a data point to a Vespa app.
+
+        :param schema: The schema that we are sending data to.
+        :param data_id: Unique id associated with this data point.
+        :param fields: Dict containing all the fields required by the `schema`.
+        :return: Response of the HTTP POST request.
+        """
+        end_point = "{}/document/v1/{}/{}/docid/{}".format(
+            self.end_point, schema, schema, str(data_id)
```

Comment:
Schema and cluster are two different things. I am not sure if it matters specifically here.

Reply:
They are different things but my (possibly wrong) understanding is that they must have the same name. At least I believe I read that in the docs and all our tutorials follow this pattern.